### ISSUE TITLE
Provide alternative to Julia dependency 

### ISSUE THREAD

**Issue by jhayase** [SewoongLab/spectre-defense#2]
It should be possible to reimplement the Julia half of the project in Python. This would be useful since not many ML researchers are familiar with Julia.

**Comment by YijiangPang** [comment]
yes, can't agree more. it will be great helpful if the Julia part can be reimplemented with Python

**Comment by jhayase** [comment]
This is something we are working on now!

**Comment by Colin-Lxy** [comment]
Waiting for your Python version. I do not know about Julia.  Thank you!

**Comment by YijiangPang** [comment]
> Waiting for your Python version. I do not know about Julia. Thank you!

You can stop waiting and find some other methods, I doubt there will be further updating.

**Comment by PythonNut** [comment]
I actually did write a version of the dkk17 covariance estimator in Python. [here is a gist](https://gist.github.com/PythonNut/5126f8ad9fde48f786e339db7be44673)

I haven't gotten around to writing a version of SPECTRE that uses this estimator, but since this is the most complicated part, I hope it can be useful.
